wafer/talks/models.py

```diff
@@ -346,15 +346,13 @@ def can_review(self, user):
 
 @reversion.register()
 class TalkUrl(models.Model):
-    """An url to stuff relevant to the talk - videos, slides, etc.
-
-       Note that these are explicitly not intended to be exposed to the
-       user, but exist for use by the conference organisers."""
+    """An url to stuff relevant to the talk - videos, slides, etc."""
 
     description = models.CharField(max_length=256)
     url = models.URLField()
     talk = models.ForeignKey(Talk, related_name='urls',
                              on_delete=models.CASCADE)
+    public = models.BooleanField(default=False)
```

Comment:
Given that the previous behaviour was equivalent to `public=True` should the default not be True?

Reply:
Sounds good to me